src/ArcGISEarth.AutoAPI.Examples/MainWindowViewModel.cs

```diff
@@ -170,95 +180,214 @@ public MainWindowViewModel()
-            ClearOutputBoxCommand = new FunctionTypeCommand(e => ExecuteFuction(FunctionType.ClearOutputBox));
-            HelpCommand = new FunctionTypeCommand(e => ExecuteFuction(FunctionType.Help));
+            SendButtonCommand = new FunctionTypeCommand(e => ExecuteFuction(FunctionType.Send));
         }
 
-        private async void ExecuteFuction(FunctionType functionType)
+        public enum SendButtonType
+        {
+            GetCamera,
+            SetCamera,
+            SetFlight,
+            AddLayer,
+            GetLayer,
+            RemoveLayer,
+            ClearLayers,
+            GetWorkspace,
+            ImportWorkspace,
+            ClearWorkspace,
+            TakeSnapshot
+        }
+
+        private SendButtonType _sendButtonType;
+        public SendButtonType SendButtontype 
         {
+            get { return _sendButtonType; }
+            set 
+            {
+                if (_sendButtonType != value)
+                {
+                    _sendButtonType = value;
+                    PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(nameof(SendButtontype)));
+                }
+            }
+        }
+
+        public string PrettyJson(string unPrettyJson)
+        {
+            var options = new JsonSerializerOptions()
+            {
+                WriteIndented = true,
+                Encoder = System.Text.Encodings.Web.JavaScriptEncoder.Create(System.Text.Unicode.UnicodeRanges.All)
+            };
+
+            if (unPrettyJson.StartsWith("{"))
+            {
+                var jsonElement = JsonSerializer.Deserialize<JsonElement>(unPrettyJson);
+                return JsonSerializer.Serialize(jsonElement, options);
+            }
+            else
+            {
+                return unPrettyJson;
+            }
+        }
+
+        string cameraExample = "{\"position\":{\"x\":-92,\"y\":41,\"z\":11000000,\"spatialReference\":{\"wkid\":4326}},\"heading\":2.3335941892764884e-17,\"tilt\":6.144145559063083e-15,\"roll\":0}";
+        string flightExample = "{\"camera\":{\"position\":{\"x\":-92,\"y\":41,\"z\":11000000,\"spatialReference\":{\"wkid\":4326}},\"heading\":2.3335941892764884e-17,\"tilt\":6.144145559063083e-15,\"roll\":0},\"duration\":2}";
+        string addLayerExample = @"{ ""URI"": ""https://www.arcgis.com/home/item.html?id=19dcff93eeb64f208d09d328656dd492"", ""target"": ""operationalLayers"", ""type"": ""PortalItem"" }";
+        string getLayerExample = "311b7317-94f8-4f80-89f2-0e3ca5e77d28";
+        string removeLayerExample = "311b7317-94f8-4f80-89f2-0e3ca5e77d28";
+        string removeLayersExample = "operationalLayers";
+        string importWorkspaceExample = @"{ ""url"": ""http://localhost:8000/workspaces/4855c0d4-9b11-4832-876b-ee3a3730dfdb.zip"", ""path"": ""C:\\Users\\Username\\Documents\\ArcGISEarth\\automation\\workspaces\\4855c0d4-9b11-4832-876b-ee3a3730dfdb.zip""}";
+        string takeSnapshotExample = @"D:\ArcGISEarth.png";
```

Comment:
Should add `private const` keyword and it better that rename `cameraExample` to `CAMERA_EXCAMPLE` ...